### PR TITLE
Added SP530e to the list of compatible controllers.

### PR DESCRIPTION
**Pull request by zalatnaicsongor**
Adds the SP530e controller to the list of WLED-compatible controllers.

### CODE REVIEW

File: docs/basics/compatible-controllers.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ In the following some third-party controllers are listed. Please use a decent an
...
...5-24v DC input, sold under BTF lighting, RGBZone, etc. Vendors all list 'BanlanX' as the supporting mobile application. Board is silk screened with 'SP530e' but no other markings. Serial pads and GPIO9 are exposed on the back-side of the board. GPIO 9 must be pulled to GND at boot to enter flashing mode. I/O configuration: LEDPIN is 'GPIO19' for addressable LEDs, BTNPIN is GPIO 3. PWM pin out for RGBWW: WW: 4, CW: 5, R: 10, G: 6 and B: 7. Level shifter used: SN74HCT245 - Warning, the flash is *encrypted* - be sure to specify the `-encrypt` flag when flashing with `esptool.py`. Each device has a unique flash encryption key, but the devs forgot to burn the 'disable UART download' efuse, which means that any non-encrypted firmware can be flashed onto the device, and the device will encrypt the firmware on the first boot. Before flashing, please back up the original firmware of the device with esptool.py `esptool.py read_flash 0 0x400000 sp530e-encrypted.bin`. Pics of board and the flashing connector here: <https://imgur.com/a/0orUKF1> -  Sample flashing command: `esptool.py write_flash --encrypt 0x0 C3_bootloader.bin 0x8000 C3_partitions_4M.bin 0x10000 WLED_0.15.0-b3_ESP32C3_4MB.bin`.
```
Comment (blazoncek) [inline docs/basics/compatible-controllers.md:88]: This is a markdown document. Please convert HTML tags to markdown.
BTW web installer has been just fixed for C3.

Comment (zalatnaicsongor) [reply docs/basics/compatible-controllers.md:88]: There's no html tag as far as I can see in the markdown document, see the <url> syntax at https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses
Is there anything else I have missed?

Comment (blazoncek) [reply docs/basics/compatible-controllers.md:88]: Sorry, my bad.

Comment (softhack007) [reply docs/basics/compatible-controllers.md:88]: If I understood correctly, using this controller only makes sense in combination with 
* https://github.com/Aircoookie/WLED/pull/4036

which was closed without merging?

Comment (blazoncek) [reply docs/basics/compatible-controllers.md:88]: Pins 18 & 19 are available for C3 if compiled with USB CDC disabled.

**Pull request merged**